src/apify_client/_utils.py

```diff
@@ -198,3 +186,18 @@ def _snake_case_to_camel_case(str_snake_case: str) -> str:
         part.capitalize() if i > 0 else part
         for i, part in enumerate(str_snake_case.split("_"))
     ])
+
+
+def _encode_key_value_store_record_value(value: Any, content_type: Optional[str] = None) -> Tuple[Any, str]:
+    if not content_type:
+        if _is_file_or_bytes(value):
+            content_type = 'application/octet-stream'
+        elif isinstance(value, str):
+            content_type = 'text/plain; charset=utf-8'
+        else:
+            content_type = 'application/json; charset=utf-8'
+
+    if 'application/json' in content_type and not _is_file_or_bytes(value) and not isinstance(value, str):
+        value = json.dumps(value, ensure_ascii=False, indent=2).encode("utf-8")
```

Comment:
Let's not indent here. It makes the resulting JSON much larger.

Reply:
I don't think it's much larger, only marginally. I've updated the HTTP client to gzip-compress the request payloads, and on the platform we store the records gzip-compressed as well, so the effect of the added spaces is negligible, but it greatly improves readability when viewing the records / actor input in the app.